tf2onnx/optimizer/onnx_graph.py

```diff
@@ -0,0 +1,379 @@
+from __future__ import division
+from __future__ import print_function
+
```

Comment:
why not just use graph.py ... most of the code in this file looks the same.

Reply:
originally I am trying to re-use graph.py, while I found more onnx graph specific things are needed, so gradually changed a lot, so I would prefer to make it a new simpler classes. I'm actually trying to make the class even more simpler.

Reply:
OnnxGraph almost have most of its member function rewritten, to 1) update output nodes/output node number, 2). get those unused node/initilizers removed automatically once set_nodes() is called.

Reply:
We want exactly one internal onnx graph / node representation. If there are changes you need in graph.py I'm sure that can be done.

Reply:
let me try